Makefile

```diff
@@ -17,6 +17,15 @@ ISO ?= $(BUILD_ROOT)/metalk8s.iso
 CALICO_CNI_PLUGIN_VERSION = 3.5.1
 CALICO_CNI_PLUGIN_BUILD = 1
 
+CONTAINER_IMAGES = \
```

Comment:
should we put this list outside the makefile ? like we did for the calico plugin sources urls

Reply:
done